### ISSUE TITLE
Overfitting Issue

### ISSUE THREAD

**Issue by agoel15006** [bentrevett/pytorch-sentiment-analysis#68]
I tried to use the LSTM architecture in a emotion classification dataset having 13 different categories of emotion. But, the model is getting overfitted. I have tried many things to prevent it but no results yet. Training accuracy reaching upto 80% while Validation accuracy moving between 30-34% only. Any suggestions?

**Comment by bentrevett** [comment]
There's no "one size fits all" solution to overfitting.

How many examples do you have? Are your classes balanced? Have you tried adding more regularisation? Are you using pretrained embeddings like GloVe or a pretrained BERT provided by the transformers library?
